docs/guides/genomics-platform/requesting-data/about-our-data.md

```diff
@@ -12,30 +12,79 @@ St. Jude Cloud hosts both raw genomic data files and processed results files:
 
 ### gVCF files
 
 We provide gVCF files produced by the [Microsoft Genomics Service][msgen]. gVCF files are derived from the BAM files produced above as called by [GATK's haplotype caller][gatk-haplotype-caller]. Today, we defer to [the official specification document][gvcf-spec] from the Broad Institute, as well as [this discussion][gvcf-diff-from-vcf] on the difference between VCF and gVCF files. For more information about how Microsoft Genomics produces gVCF files or any other questions regarding data generation, please refer to [the official Microsoft Genomics whitepaper][msgen-whitepaper].
 
 ### Somatic VCF files
 
-Somatic VCF files contain HG38 based SNV/Indel variant calls from the St. Jude somatic variant analysis pipeline as follows. Broadly speaking:
+Somatic VCF files available on St. Jude Cloud have been generated when possible for tumor samples where a matching normal sample has also been sequenced. These include samples from the Pediatric Cancer Genome Project (PCGP), Clinical Pilot, and Genomes 4 Kids (G4K) [cohorts][pediatric-cancer-cohorts]. These somatic VCF files contain HG38 coordinate SNV/Indel variant calls generated by the St. Jude somatic variant analysis pipeline outlined below:
 
-1. Reads were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
+1. Whole genome (WGS) and/or Whole exome (WES) sequencing data were aligned to HG19 using [bwa backtrack][bwa] (`bwa aln` + `bwa sampe`) using default parameters.
 2. Post processing of aligned reads was performed using [Picard][picard] `CleanSam` and `MarkDuplicates`.
-3. Variants were called using the [Bambino][bambino-paper] variant caller (you can download Bambino [here][bambino-download] or by navigating to the [Zhang Lab page][bambino-program] where the  "Bambino package" is listed as a dependency under the CONSERTING section).
-4. Variants were post-processed using an in-house post-processing pipeline that cleans and annotates variants. This pipeline is not currently publicly available.
-5. Variants were manually reviewed by analysts and published with [the relevant Pediatric Cancer Genome Project (PCGP) paper][pcgp-landing-page].
-6. Post-publication, variants were lifted over to HG38 (the original HG19 coordinates are stored in the `HG19` INFO field.).
+3. Variants were called using the [Bambino][bambino-paper] variant caller (you can download Bambino [here][bambino-download]).
+4. If RNA-Sequencing was performed on a tumor sample, we assess the presence of any called variants in a StrongARM aligned RNA-Seq BAM file.
```

Comment:
We should add a citation to Gang's HGG paper that describes the StrongARM method: https://www.nature.com/articles/ng.2938.

Reply:
I was told we never published the StrongARM method, so good to know we have a link for it. Adding it now.